css/styles.css

```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: crimson;
+  text-align: center;
+}
```

Comment:
- [ ] As required for the project, kindly ensure that your `css/style.css` file includes one class with styles for your header i.e. you need to add a class in your `h1` tag from your `index.html` file and use that class in your ``css/style.css`` file for styling instead of using the `h1` tag. Example:
```css
.heading {
  color: crimson;
  text-align: center;
}
```